### PR TITLE
Fixed X64Fact tests

### PR DESCRIPTION
**Pull request by mstfbl**
Fixed X64Fact tests with x86 specific baselines added

Currently obtaining baselines from x86 builds from CI

### CODE REVIEW

**Comment by harishsk** [comment]
Yes, I noticed that too when I was debugging earlier. 

Can you please do me a favor? 

Now that we have gone towards different baselines, I think it would be good to be able to easily track which tests are using configuration specific baselines. Can you please add some code in BaseTestBaseline.cs to log whenever we use a configuration specific baseline? I just realized that this is something we should do. 


---
In reply to: [399621483](https://github.com/dotnet/machinelearning/pull/5057#issuecomment-399621483) [](ancestors = 399621483)

**Comment by harishsk** [comment]
You can do it as a separate PR if you wish.

File: test/Microsoft.ML.TestFramework/BaseTestBaseline.cs [hunk 1]
```diff
@@ -238,6 +239,15 @@ protected void Log(string fmt, params object[] args)
             Output.WriteLine(fmt, args);
         }
 
+        protected void LogConfigurationSpecificBaseline(string baselineConfigDir)
```
Comment (harishsk) [inline test/Microsoft.ML.TestFramework/BaseTestBaseline.cs:242]: So this is going to log for every file comparison? Can this be done only once per test?

File: test/Microsoft.ML.Predictor.Tests/TestPredictors.cs [hunk 2]
```diff
@@ -187,7 +187,7 @@ public void MulticlassSdcaTest()
         /// <summary>
         /// Multiclass Logistic Regression test with a tree featurizer.
         /// </summary>
-        [X64Fact("x86 output differs from Baseline")]
```
Comment (frank-dong-ms) [inline test/Microsoft.ML.Predictor.Tests/TestPredictors.cs:190]: >X64Fact [](start = 9, length = 7)

can we remove X64Fact now?

Comment (harishsk) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs:190]: As @mstfbl pointed out above, I think there are still around 4 tests that are causing test process crashes in x86. They need investigation.

Comment (frank-dong-ms) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs:190]: I see, thanks

---
In reply to: [414314991](https://github.com/dotnet/machinelearning/pull/5057#discussion_r414314991) [](ancestors = 414314991)

**Comment by mstfbl** [comment]
I will make a separate PR for logging when a configuration specific benchmark is utilized.

**Pull request merged**